Fix bug in Aruba reporting where methods are called with parameters that have been removed
diff --git a/lib/aruba/reporting.rb b/lib/aruba/reporting.rb
index a588ce0..9467e7c 100644
--- a/lib/aruba/reporting.rb
+++ b/lib/aruba/reporting.rb
@@ -23,8 +23,8 @@ def pygmentize(file)
         pygmentize.run! do |p|
           exit_status = p.stop(false)
           if(exit_status == 0)
-            p.stdout(false)
-          elsif(p.stderr(false) =~ /no lexer/) # Pygment's didn't recognize it
+            p.stdout
+          elsif(p.stderr =~ /no lexer/) # Pygment's didn't recognize it
             IO.read(file)
           else
             STDERR.puts "\e[31m#{p.stderr} - is pygments installed?\e[0m"
